### DISCUSSION TITLE
Send Keystroke on macOS - Ctrl + Shift + 6

### DISCUSSION THREAD

**Discussion by scopapa** [isontheline/pro.webssh.net#1147] [Q&A] [answered]
Hello, how do I send the keystroke Ctrl + Shift + 6 in an SSH session? This is to set a mark point in nano so I can bulk modify text. This works in the terminal app but I'd like to get it working (or figure out an alternative) in WebSSH. Thank you.

**Comment by isontheline** [comment, marked as answer]
Hello @scopapa 👋

Yet this keystroke is not mapped inside WebSSH but I could add it

**Reply by scopapa** [reply]
Thank you so much! Love the client!

**Reply by isontheline** [reply]
Just created an issue about your request : https://github.com/isontheline/pro.webssh.net/issues/1149
I will try to work on it as soon as possible
